### PR TITLE
docs: Add link to documentation & improvements from MADR 0001

### LINKED ISSUE #12: CI/CD: run stress tests against nightly container

### LINKED ISSUE #20: docs: Migrate current documentation to new docs
## What needs to be done

Migrate the docs here:

- https://github.com/aquarist-labs/s3gw-core/tree/main/docs
- https://github.com/aquarist-labs/s3gw-charts/blob/main/charts/s3gw/README.md

to the documentation solution implemented in this task: https://github.com/orgs/aquarist-labs/projects/5/views/12

Since the content is related to two separate projects (the s3gw core & charts), the content should be grouped in specific sections:

- s3gw core
- charts
- UI (no content here yet, but we'll have to consider this section)
- sfs -> This is a section that may be also needed in the future

The current links to these documents in the readme files also need to be updated to the new structure.

**Important**
MADRs also need to get migrated.

## Why it needs to be done

As part of the effort in improving our documentation.

## Acceptance Criteria

The user is able to navigate through the different pages in the documentation.

### PR DESCRIPTION
**Pull request by jhmarina**
This PR:
- Fixes #20
- Updates main readme to only include quickstart info, license & a link to the documentation
- Implements updates proposed in ADR 0001: https://github.com/aquarist-labs/s3gw/blob/main/docs/decisions/0001-reorganize-documentation.md

Signed-off-by: Jesus Herman-Marina <jherman@suse.com>

### CODE REVIEW

File: README.md
Comment (m-ildefons) [inline README.md]: This paragraph seems to be an old remnant of the s3gw-core repo and doesn't fit particularly well into the current structure.

**Review by m-ildefons — CHANGES_REQUESTED** [review]
Found an out of date paragraph adjacent to the changes. Otherwise this looks very good.

File: README.md
Comment (jhmarina) [reply README.md]: Updated

File: README.md
Comment (jecluis) [inline README.md]: this removes the indentation from the license notice. Unsure if it was meant, but looks weird this way :shrug:

File: README.md
Comment (jecluis) [inline README.md:28]: Are we removing the roadmap here on purpose? Where will it be living otherwise?

**Review by jecluis — COMMENTED** [review]
few comments/questions, not blocking merge.

File: README.md
Comment (jhmarina) [reply README.md:28]: yes, it'll live in the documentation here: https://s3gw-docs.readthedocs.io/en/latest/ - once this PR is merged, I'll be able to generate the new documentation

File: README.md
Comment (jhmarina) [reply README.md]: I need to review my linter rules 😄. I sorted it out

**Review by m-ildefons — DISMISSED** [review]
LGTM

**Comment by jecluis** [comment]
could we have the commit you want to merge rebased on top of `main` instead of having a merge commit in the PR? :)

**Comment by jhmarina** [comment]
> could we have the commit you want to merge rebased on top of `main` instead of having a merge commit in the PR? :)

Apologies @jecluis , I didn't see your comment when I hit the Merge button. I'll be sure I rebase on top of `main` instead next time

**Pull request merged**
